docs/source/index.rst

```diff
@@ -139,10 +139,21 @@ Additional back-ends could be implemented if the need arose.
 |                                                        |                                                      |
 |    ARM diagnostics monthly diurnal cycle of cloud plot |    ARM diagnostics convection onset statistics plot  |
 +--------------------------------------------------------+------------------------------------------------------+
+| .. figure:: _static/index/fig21.png                    | .. figure:: _static/index/fig22.png                  |
+|    :align: center                                      |    :align: center                                    |
+|    :target: _static/index/fig21.png                    |    :target: _static/index/fig22.png                  |
+|                                                        |                                                      |
+|    Tropical Cyclone Track Density                      |    Annual Cycle Zonel Mean plot                      |
++--------------------------------------------------------+------------------------------------------------------+
+| .. figure:: _static/index/fig23.png                    | .. figure:: _static/index/fig24.png                  |
+|    :align: center                                      |    :align: center                                    |
+|    :target: _static/index/fig23.png                    |    :target: _static/index/fig24.png                  |
+|                                                        |                                                      |
+|    Tropical Cyclone frequency per basin                |    Per-basin Tropical Cyclone frac seasonal cycle    |
++--------------------------------------------------------+------------------------------------------------------+
 
 The above plots and more can be found
-`here <https://portal.nersc.gov/cfs/e3sm/zhang40/tutorials/run_v230_allsets/viewer/>`_.
-ARM diagnostics plots can be found `here <https://web.lcrc.anl.gov/public/e3sm/e3sm_diags_test_data/unit_test_complete_run/expected/all_sets/viewer/arm_diags/>`_.
+`here <https://portal.nersc.gov/cfs/e3sm/zhang40/tutorials/run_latest_allsets/viewer/>`_.
```

Comment:
I'm getting a `404 Not Found` for this URL

Reply:
yeah, it is expected. I will have a latest allsets run at a later time when the run script example is updated:)

Reply:
I rolled back to the old link, and will update after the latest result is updated.